src/init/features/apphosting/index.ts

```diff
@@ -132,6 +144,96 @@ export async function doSetup(
   logSuccess(`Your site is now deployed at:\n\thttps://${backend.uri}`);
 }
 
+/**
+ *
+ * @param projectId user's projectId
+ * @param webAppName (optional) name of web app
+ * @return app name and app id
+ */
+async function getOrCreateWebAppId(
+  projectId: string,
+  webAppName: string | null,
+  backendId: string,
+): Promise<string[]> {
```

Comment:
how about instead of Promise<string[]>, type WebApp = [string, string];

or even better, type WebApp = {name: string, id: string}

Reply:
yep this is much better!